xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-test/xwiki-platform-administration-test-pageobjects/src/main/java/org/xwiki/administration/test/po/ThemesAdministrationSectionPage.java

```diff
@@ -64,82 +70,134 @@ public ThemesAdministrationSectionPage()
         super("Themes");
     }
 
+    private List<WebElement> getThemeOptions(WebElement themeInput) {
```

Comment:
code style issue with the curly bracket. Do you have the xwiki code style loaded in IntelliJ IDEA? See https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HConfiguringyourIDEtousetheXWikicodestyle

Maybe you wrote it by hand and you didn't reformat it through the IDE.